core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.component.html

```diff
@@ -17,11 +17,11 @@
     <form class="form-horizontal" ng-submit="$ctrl.submitViewContributorUsersForm($ctrl.formData.viewContributionReviewers)">
       <legend class="text-center">View Contributor Dashboard Users</legend>
       <div class="d-xl-inline-flex form-group">
-        <label class="col-md-4 col-lg-4 col-sm-4 col-form-label" for="label-target-view-reviewer-method">
+        <label class="col-md-4 col-lg-4 col-sm-4 col-form-label" for="view-contributor-method-select">
```

Comment:
```suggestion
        <label class="col-md-4 col-lg-4 col-sm-4 col-form-label" for="view-contributor-filter-method-select">
```
For clarity in the tests. Ditto below.

Reply:
Done